core/src/main/scala/cats/instances/map.scala

```diff
@@ -76,7 +76,7 @@ trait MapInstances extends cats.kernel.instances.MapInstances {
         fa.flatMap { case (k, a) => f(a).get(k).map((k, _)) }
 
       def unorderedFoldMap[A, B: CommutativeMonoid](fa: Map[K, A])(f: (A) => B) =
-        fa.foldLeft(Monoid[B].empty) { case (b, (k, a)) => Monoid[B].combine(b, f(a)) }
+        fa.foldLeft(Monoid[B].empty) { case (b, (_, a)) => Monoid[B].combine(b, f(a)) }
```

Comment:
nit: this should be:

```scala
Monoid[B].combineAll(fa.iterator.map { case (_, a) => f(a) })
```
so the monoid can use interior mutability.

Reply:
✅